modules/distr-tracing-tempo-object-storage-setup-ibm-storage.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-object-storage-setup-ibm-storage_{context}"]
+= Setting up {ibm-cloud-title} Object Storage
+
+You can set up {ibm-cloud-title} Object Storage on the command line.
+
+.Prerequisites
+
+* You have installed the latest version of the OpenShift CLI. For more information, see "Getting started with the OpenShift CLI" in _CLI tools_.
+
+* You have installed the latest version of the {ibm-cloud-title} Command Line Interface (AWS CLI). For more information, see "Getting started with the {ibm-cloud-title} CLI" in _{ibm-cloud-title} Docs_.
+
+* You have an {ibm-cloud-title} Platform account and ordered an {ibm-cloud-title} Object Storage plan and created an instance of {ibm-cloud-title} Object Storage. For more information, see "Choosing a plan and creating an instance" in _{ibm-cloud-title} Docs_.
+
+.Procedure
+
+====
+
+. Fetch your credentials from the cluster deployed on {ibm-cloud-title}:
++
+[source,terminal]
+----
+export IBMCLOUD_API_KEY=$(oc get secrets ibmcloud-creds -n kube-system -o json | jq -r '.data.apiKey' | base64 -d)
+----
++
+. Log in to {ibm-cloud-title} by using {ibm-cloud-title} CLI:
++
+[source,terminal]
+----
+$ ibmcloud login -r us-east 
+----
++
+. Install the `cloud-object-storage` and `infrastructure-service` plugins:
++
+[source,terminal]
+----
+$ ibmcloud plugin install cloud-object-storage \
+  && \
+  ibmcloud plugin install infrastructure-service
+----
++
+. Create a resource group:
++
+[source,terminal]
+----
+$ ibmcloud resource group-create tempo-tracing \
+  && \
+  ibmcloud target -g tempo-tracing
+----
++
+. Create a service instance for the object store:
++
+[source,terminal]
+----
+$ ibmcloud resource service-instance-create -d premium-global-deployment \
+  <tempo_bucket> cloud-object-storage standard global \
+  && \
+  export IBM_SERVICE_INSTANCE_ID=$(ibmcloud resource service-instance <tempo_bucket> -o json | jq -r '.[0].crn')
+----
++
+. Create an object store bucket:
```

Comment:
@IshwarKanse and @rubenvp8510, can I add this as a prerequisite?

Reply:
No

Reply:
Actually, we can mention that create a IBM cloud object store bucket before following the next steps.

Reply:
TY. Mentioning.